<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetShowHistoryCell</class>
 <widget class="QWidget" name="WidgetShowHistoryCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>12</number>
   </property>
   <item row="1" column="1">
    <widget class="QLabel" name="labelDateTime">
     <property name="text">
      <string notr="true">labelDateTime</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelLink">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">labelLink</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="iconFile">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources.qrc">:/icons/file-audio.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
